coreprofile/10/_index.md

```diff
@@ -1,37 +1,66 @@
 ---
 title: "Jakarta EE Core Profile 10 (under development)"
-date: 2021-04-07
-summary: "Release of the Jakarta EE Core Profile 10"
+date: 2021-08-31
+summary: "Release of the Jakarta EE 10 Core Profile"
 ---
-The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifically targeted at smaller runtimes.
+The Jakarta EE Core Profile defines a profile of the Jakarta EE Platform targeted at cloud native microservice
+applications with support for build time/ahead of time compilation.
 
-The goals for this release are:
-1. To provide a profile that contains a set of Jakarta EE Specifications targeting smaller runtimes suitable for microservices and ahead-of-time compilation.
-2. The TCK(s) for the Core profile should be possible to run independently of the Full Platform TCK.
-3. Java SE version will be aligned with the Platform Project's decision regarding Java SE versions.
-
-
-* [Jakarta EE Core Profile 10 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/core-profile-10)
+* [Jakarta Core Profile 10 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/core-profile-10)
+  * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+* [Jakarta Core Profile 10 Specification Document](./jakarta-coreprofile-spec-10.pdf) (PDF)
+* [Jakarta Core Profile 10 Specification Document](./jakarta-coreprofile-spec-10.html) (HTML)
+* [Jakarta Core Profile 10 Javadoc](./apidocs)
+* [Jakarta Core Profile 10 TCK](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
+  * [TBD]()
+  
+# Jakarta EE 10 Schedule
+* [Jakarta EE 10 Schedule](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10#jakarta-ee-10-schedule)
 
+# Compatible Implementations
+* [Jakarta EE 10 Compatible Implementations](https://jakarta.ee/compatibility/#tab-10)
 
 # Ballots
 
 ## Plan Review
 
+The Specification Committee Ballot concluded successfully on 2021-08-19 with the following results.
```

Comment:
Is this actually correct? The "Creation and Plan Review" was conducted in May. It was not a part of the ballot for Platform and Web Profile.

Reply:
I have corrected that.